.gitignore

```diff
@@ -763,7 +763,7 @@ MigrationBackup/
 
 # End of https://www.toptal.com/developers/gitignore/api/windows,macos,linux,python,react,pycharm,emacs,vim,visualstudio,visualstudiocode
 
-/docs/source/api/*.rst
+/docs/api/*.rst
```

Comment:
Need to remove, we shouldn't be ignoring anything for the api docs rst files now